first-analysis-steps/lhcb-dev.md

```diff
@@ -211,82 +245,75 @@ When pushing to a branch in a project in GitLab you will see a message like:
 If the aim of the commit was to change the ouput, e.g. because you increased
 the track reconstruction efficiency by a factor of two, mention it in the
-merge request description, such that the manager of the affected project can
+merge request description, such that the maintainers of the affected project can
 update the reference file.
 
-{% endcallout %} 
+Often it is useful to work directly from nightly builds (e.g. to debug an issue on another platform).
+You can use a nightly build version of a project with:
+
+```bash
+lb-dev -c x86_64-centos7-gcc9-opt --nightly lhcb-run2-patches DaVinci/run2-patches
+```
+
+A more detailed description of the command is found at [SoftwareEnvTools](https://twiki.cern.ch/twiki/bin/view/LHCb/SoftwareEnvTools).
+
+{% endcallout %}
 
 If you want to take a look the source code, without checking it out, you can
 easily access the repository through the [GitLab web
 interface](https://gitlab.cern.ch/lhcb). This website also provides search
 functionality, but the output is not always easy to read, especially if it
-returns many hits. To search a project much quicker, you can use `Lbglimpse`.
+returns many hits. To search a project much quicker, you can use `lb-glimpse`.
 It allows you to search for a given string in the source code of a particular
-LHCb project.
+LHCb project (and all its dependencies).
 
 ```bash
-Lbglimpse "PVRefitter" DaVinci v45r1
+lb-glimpse "PVRefitter" DaVinci/v45r1
 ```
 This works with every LHCb project and released version. Since it's a shell
 command, you can easily process the output using `less`, `grep`, and other
 tools.
 
 To get an idea of how a certain component of the LHCb software works, you can
 also access the doxygen documentation. One set of doxygen web pages is
-generated for several related projects, and is linked in all the projects web
+generated for several related projects, and is linked from the projects web
 sites, like [for DaVinci](http://lhcbdoc.web.cern.ch/lhcbdoc/davinci/).
-See also the [LHCb Computing web page](http://lhcb.web.cern.ch/lhcb/computing/) for a
+For example, the doxygen documentation for DaVinci v45r1 is
+[here](https://lhcb-doxygen.web.cern.ch/lhcb-doxygen/davinci/v45r1/index.html).
+See also the [LHCb Computing web page](https://lhcb.web.cern.ch/computing/) for a
 list of projects.
 
 {% callout "Working with a full project checkout" %}
 
-The `lb-git` commands are not strictly necessary, but they're very convenient
+The `git lb-*` commands are not strictly necessary, but they're very convenient
 if you just want to quickly edit one package. Otherwise you'd have to build
 the entire project in which the package is residing, instead of using the
 precompiled version. However, if you develop across multiple packages, or
 want to use more sophisticated `git` commands, nothing prevents you from
-checking out an entire project – just don't be surprised if it takes O(hours)
+checking out an entire project – just don't be surprised if it takes hours
```

Comment:
Maybe we should be more specific here? And say it takes hours on lxplus but if you a have some multi-core machines available it takes less than an hour and only for the first time?

Reply:
:+1: